Pspcl/Pspcl.Core/Domain/Stock.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Pspcl.Core.Domain
 {
-    public class Stock
+    public class Entity
```

Comment:
What is the need of doing this ?

Reply:
it is happend due to property name on view model and database entity are not matching so automapper used this default naming convention to map the properties

Reply:
So, you gona change it to Entity for all entity classes ?